example/spec/parallel/lzardsBackup/LzardsBackupSpec.js

```diff
@@ -134,8 +143,107 @@ describe('The Lzards Backup Task ', () => {
 
   it('has the expected backup information', () => {
     const backupStatus = JSON.parse(lzardsBackupOutput.Payload).meta.backupStatus;
+    const returnedCollection = JSON.parse(lzardsBackupOutput.Payload).meta.collection;
```

Comment:
The `meta.collection` is in the input/output cumulus message, and not added by the task.  Please add collectionId to the task output so it's available in backupStatus